lib/tutorials/en_US/getting-started.md

```diff
@@ -18,23 +19,25 @@ The most basic server looks like the following:
 
 const Hapi = require('hapi');
 
-const server = new Hapi.Server();
-server.connection({ port: 3000, host: 'localhost' });
+const server = Hapi.server({ port: 3000, host: 'localhost' });
 
-server.start((err) => {
-
-    if (err) {
-        throw err;
-    }
+const init = async () => {
+    await server.start();
     console.log(`Server running at: ${server.info.uri}`);
+};
+
+init().catch((err) => {
```

Comment:
I think the preferred approach is to attach a `onUncaughtException` handler and avoid use of the promise API directly.